packages/ia-components/scripts/version.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const { version: NEW_VERSION } = require('../package.json');
+
+async function main() {
+  // Update the changelog
+  const old_changelog = fs.readFileSync('CHANGELOG.md').toString();
+  const new_changelog = `# ${NEW_VERSION}\n<!-- Write here... -->\n\n` + old_changelog;
```

Comment:
SUGGESTIONS: 1) Use camelcase for old_changelog and new_changelog. 2) Since `<!-- Write here... -->` appears twice, DRY it up by putting in a constant to reduce connaissance.